app/src/main/kotlin/io/element/android/x/ElementXApplication.kt

```diff
@@ -36,5 +38,6 @@ class ElementXApplication : Application(), DaggerComponentOwner {
             initializeComponent(TracingInitializer::class.java)
         }
         logApplicationInfo()
+        bindings<AppBindings>().cacheCleaner().clearCache()
```

Comment:
I know I said this didn't have to be an initializer too, but looking at the previous code it seems a bit out of place if it's not 🤔 . Would it take you much time to migrate to using an initializer? If so, we can leave it as is.

Reply:
Done in: https://github.com/vector-im/element-x-android/pull/1807/commits/47111b9a535007449f848cf09e07f3badcc833b4